### PR TITLE
Rights Flow Removal

### LINKED ISSUE #64: Bullet point icon size too large
Small thing but I noticed the bullet points on one of the scenarios pages are too big.

Simulator Screen Shot - iPhone 11 Pro - 2020-09-10 at 21 00 22

### PR DESCRIPTION
**Pull request by knarangerel**
- Removes two screens from onboarding flow and replaces with a new screen 

- Updates emergency screen to reflect new flow 

- Resolves #64 and errors in scenario screens created from replacing `TouchableOpacity` with `TouchableHighlight` in `Card` component 

- Adds an empty `View` (with height) at the bottom of the home screen so that the illustration is still fully visible with large text sizes if you scroll down

Notes: 
1. I'm not sure if I have the right modal on the emergency screen. The wireframes were not updated. 
2. I considered removing `ProgressCircles` component altogether because we're not using it anymore, but left it just in case.

Screen Shot 2020-09-15 at 20 24 17
Screen Shot 2020-09-15 at 20 24 39
Screen Shot 2020-09-15 at 20 24 48

### CODE REVIEW

File: js/screens/EmergencyScreen.js [hunk 1]
```diff
@@ -11,202 +12,152 @@ import PropTypes from 'prop-types';
...
+            title={t('learn_more')}
           />
-          <View style={{ flexGrow: 1 }} />
         </ScrollView>
-
-        {/* ABSOLUTE MODALS */}
+        {/* MODAL */}
         <InfoModal
           isVisible={InfoModalVisible}
           setModalVisible={setInfoModalVisible}
```
Comment (JacobJaffe) [inline js/screens/EmergencyScreen.js:127]:
```suggestion
          onClose={() => setInfoModalVisible(false)}
```
(See above comment on the `InfoModal`)

File: js/screens/rights/RightsOverviewScreen.js [hunk 2]
```diff
@@ -82,6 +82,7 @@ export default function RightsOverviewScreen({ navigation }) {
         />
         <View style={{ height: 8 }} />
         <DisclaimerCard />
+        <View style={{ height: 180 }} />
```
Comment (JacobJaffe) [inline js/screens/rights/RightsOverviewScreen.js:85]: What is this for? I think this will force unnecessary scrolling on small devices

**Review by JacobJaffe — COMMENTED** [review]
Good stuff! A few suggestions & questions on intended behavior on some of the changes

File: js/components/Card.js [hunk 3]
```diff
@@ -37,7 +37,6 @@ const styles = StyleSheet.create({
     padding: 12,
   },
   cardUnderlay: {
-    backgroundColor: colors.primaryLight,
```
[the remark this answers is not in the document]
Comment (knarangerel) [reply js/components/Card.js:40]: The underlay color is specified as a prop to `TouchableHighlight`. I'll get rid of `cardUnderlay` styling element completely. It's kind of confusing.

File: js/screens/EmergencyScreen.js [hunk 4]
```diff
@@ -11,202 +12,152 @@ import PropTypes from 'prop-types';
...
+
+const InfoSection = () => {
+  const { t } = useTranslation();
+  return (
+    <Card style={styles.infoCard}>
+      <>
+        <Text style={[textStyles.h3, { paddingVertical: 6 }]}>
+          {t('call_hotline_if')}
+        </Text>
+        <Text style={[textStyles.body1, styles.infoText]}>
```
[the remark this answers is not in the document]
Comment (knarangerel) [reply js/screens/EmergencyScreen.js:45]: I've tried the ways you suggested, but I'm finding it hard to achieve the exact padding necessary between the text components, so I'm leaving it as is for now.

File: js/screens/EmergencyScreen.js [hunk 5]
```diff
@@ -11,202 +12,152 @@ import PropTypes from 'prop-types';
...
+      <Text style={textStyles.h1}>{t('emergency_hotline')}</Text>
+    </View>
+  );
+};
+
+const InfoSection = () => {
+  const { t } = useTranslation();
+  return (
+    <Card style={styles.infoCard}>
+      <>
```
[the remark this answers is not in the document]
Comment (knarangerel) [reply js/screens/EmergencyScreen.js:41]: `TouchableHighlight` seemed to require only one node to render (many uses of `Card` got errors after our switch from `TouchableOpacity`), and adding fragments around the content components seemed to do the trick. I figured the fragment was a way to package multiple components together into one node. Is there a specific reason we want an explicit parent node, or is it more about fragment usage?

File: js/screens/EmergencyScreen.js [hunk 6]
```diff
@@ -11,202 +12,152 @@ import PropTypes from 'prop-types';
...
-                darkMode
-                onPress={() => setRightsCardVisible(!RightsCardVisible)}
-                title={t('rights_card')}
-              />
             </View>
           </View>
-          <View style={{ flexGrow: 1, maxHeight: 100 }} />
           <HelpButton
             centered
             onPress={() => setInfoModalVisible(!InfoModalVisible)}
```
[the remark this answers is not in the document]
Comment (knarangerel) [reply js/screens/EmergencyScreen.js:120]: You're right!

File: js/screens/EmergencyScreen.js [hunk 7]
```diff
@@ -11,202 +12,152 @@ import PropTypes from 'prop-types';
...
+        </Text>
+        <Text style={[textStyles.body1, styles.infoText]}>
+          {t('you_are_being_patrolled')}
+        </Text>
+      </>
+    </Card>
+  );
+};
+
+const InfoModal = ({ isVisible, setModalVisible }) => {
```
[the remark this answers is not in the document]
Comment (knarangerel) [reply js/screens/EmergencyScreen.js:59]: Good catch! I'll change the component and its use!

File: js/screens/rights/RightsOverviewScreen.js (hunk 2 above)
Comment (knarangerel) [reply js/screens/rights/RightsOverviewScreen.js:85]: The `View` is the exact height of the illustration. My intent was to ensure that the illustration is visible when you scroll to the bottom with large text sizes. You can see the difference with and without below when both screens are scrolled all the way to the bottom. The extra `View` also didn't affect functionality with smaller text sizes. I'm really not sure if this is the way to go about it, and I'm open to better ways to go about it!

Simulator Screen Shot - iPhone 11 Pro - 2020-09-23 at 13 34 05

Simulator Screen Shot - iPhone 11 Pro - 2020-09-23 at 13 33 57

File: js/components/Card.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (JacobJaffe) [reply js/components/Card.js:40]: Ooo, good catch

File: js/screens/EmergencyScreen.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (JacobJaffe) [reply js/screens/EmergencyScreen.js:41]: My understanding is that fragment essentially gets removed from the tree, so it gets decoupled to many nodes, so I'm surprised this works correctly with something that expects a single node. If it works that's all cool

something to check cross platform on, this is the kinda thing that could have wonky behavior on different devices

File: js/screens/EmergencyScreen.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (JacobJaffe) [reply js/screens/EmergencyScreen.js:45]: Ah, using the explicit text for these to be different in layout positioning makes sense

File: js/screens/rights/RightsOverviewScreen.js (hunk 2 above)
Comment (JacobJaffe) [reply js/screens/rights/RightsOverviewScreen.js:85]: Gotchya. could that be more explicit in the code, by defining an `IMAGE_HEIGHT` that is shared between the two?

**Review by JacobJaffe — APPROVED** [review]

**Review by JacobJaffe — APPROVED** [review]
still lgtm

**Pull request merged**